Initial English User Manual (#1)
mattogodoy/open-rally-computer: docs/manual/manual_english.md

- **Down**: Recall partial distance down (reduce). If the distance accuracy is set to 1 decimal, subtract 100 meters. If it is set to 2 decimal places, subtract 10 meters. Keeping the button pressed will quickly zoom out to make changes of great distances.
- **Selection**: If pressed briefly, it switches to the next screen. If it stays pressed resets the partial distance to 0. This **DOES NOT** affect the total distance.

### Speed

On this screen you can see:
- The speed at which you are currently moving.
- The average speed since the last reset carried out from the configuration menu.
**NOTE**: This average includes the time the motorcycle is not moving. This is especially useful to know your real average the regularity rallies.
- The maximum speed reached since the last reset carried out from the settings menu setting. This value is useful to avoid being penalized in rallies where speed maximum is limited.
**Controls**
On this screen, the remote control buttons fulfill the following functions:

> - **Up**: If the QuickView functionality is enabled, displays the Odometer screen while holding down the button.
> 
> 
> `` ''
> - Bottom: If the QuickView functionality is enabled, displays the Odometer screen while holding down the button.
> - Selection: Short press switches to next screen. If it stays pressed it accesses the configuration screen.
> `` ''

Reviewer: ```suggestion
- **Up**: If the QuickView functionality is enabled, displays the Odometer screen while holding down the button.
- **Down**: If the QuickView functionality is enabled, displays the Odometer screen while holding down the button.
- **Selection**: Short press switches to next screen. Long press accesses the configuration screen.
```

### Travel time

> On this screen you can see the time elapsed since the last reset performed since the settings menu, including the time the motorcycle has been stopped. This is especially useful for regularity rallies.

Reviewer: ```suggestion
In this screen you can see the time elapsed since the last total distance reset performed from the settings menu, including the time the motorcycle has been stopped. This is especially useful for regularity rallies.
```

**Controls**

> On this screen, the remote control buttons fulfill the following functions:

Reviewer: ```suggestion
In this screen, the remote control buttons fulfill the following functions:
```

- **Up**: If the QuickView functionality is enabled, displays the Odometer screen while holding down the button.
- **Down**: If QuickView functionality is enabled, displays the Odometer screen while holding down the button.

> - **Selection**: If pressed briefly, it switches to the next screen. If it stays pressed it accesses the configuration screen.

Reviewer: ```suggestion
- **Selection**: If pressed briefly, it switches to the next screen. Long press accesses the configuration screen.
```

### Course

Indicates the current heading in degrees, being 0º North and 180º South.
**NOTE**: Heading is calculated by drawing vectors between GPS location points, so this value is not valid if the motorcycle is not moving.
**Controls**

> On this screen, the remote control buttons fulfill the following functions:

Reviewer: ```suggestion
In this screen, the remote control buttons fulfill the following functions:
```

- **Up**: If the QuickView functionality is enabled, displays the Odometer screen while holding down the button.
- **Down**: If QuickView functionality is enabled, displays the Odometer screen while holding down the button.

> - **Selection**: If pressed briefly, it switches to the next screen. If it stays pressed it accesses the configuration screen.

Reviewer: ```suggestion
- **Selection**: If pressed briefly, it switches to the next screen. Long press accesses the configuration screen.
```

> ## Setting

Reviewer: ```suggestion
## Settings
```

> This is the screen from which the **Baja Pro** is configured.

Reviewer: ```suggestion
This is the screen from where the **Open Rally Computer** is configured.
```

### Access the configuration menu

> To access the configuration menu, I am **stating at any screen other than the Odometer**, press and hold the **Select** button (rear button of controller) for a few seconds.

Reviewer: ```suggestion
To access the configuration menu, **while at any screen other than the Odometer**, press and hold the **Select** button (rear button of controller) for a few seconds.
```

### Navigate within settings

> To select options within the setup menu using the knob:

Reviewer: ```suggestion
To select options within the setup menu using the remote:
```

- **Up**: Upper front button
- **Lower**: Lower front button
- **Select**: Back button
- **Cancel**: Press and hold the back button
- **Exit menu**: Press and hold the back button

### Reset trip

Use this option before starting each stage of a rally. This resets:
- Total distance
- Partial distance
- Average speed
- Maximum speed
- Travel time

### Language

> The **Baja Pro** can be configured in the following languages:

Reviewer: ```suggestion
The **Open Rally Computer** can be configured in the following languages:
```

- Spanish
- English
Select the one that best suits your needs.

### Accuracy

> The accuracy level of the **Baja Pro** can be set between:

Reviewer: ```suggestion
The accuracy level can be set between:
```

> - **One decimal**: Distances show increments every 100 meters. Upon landing, will add or subtract 100 meters to the partial distance.

Reviewer: ```suggestion
- **One decimal**: Distances show increments every 100 meters. When manually adjusting, it will add or subtract 100 meters to the partial distance.
```

> - **Two decimal places**: Distances show increments every 10 meters. Upon landing, will add or subtract 10 meters to the partial distance.

Reviewer: ```suggestion
- **Two decimal places**: Distances show increments every 10 meters. When manually adjusting, it will add or subtract 10 meters to the partial distance.
```

### Units

> The **Baja Pro** unit system can be configured in:

Reviewer: ```suggestion
The units system can be configured as:
```

- Metric (km / h, mm)

> - Imperial (mph / in)

Reviewer: ```suggestion
- Imperial (mph, in)
```

This option affects distance measurements. In the **Metric** system **kilometers** are used and in the
**Imperial** system **miles** are used.
**NOTE**: Changing this option resets all time, speed and distance values.

### Time zone

Allows you to select the time zone you are in to obtain a correct value for the current time.
The range of **UTC** values ​​is between **- 12** and **+ 14.**

### Backlight

Select how bright the screen is.
The range of values ​​is from **0** (light off) to **10** (maximum intensity). This is especially useful for low-light or night stages where the brightness of the display may affect the visibility of the pilot.

### Quickview

> This functionality allows you to quickly and temporarily return to the home screen.
> **Odometer** when you are on any other screen.
> If this option is activated, whenever you are on a screen other than **Odometer**, at holding down the upper or lower button on the remote will temporarily display the **Odometer** display. Releasing the button on the controller will return to the screen where you you found previously.
> This is especially useful for sections where you need to pay special attention to the **Heading** or **Speed ​​**, but from time to time you need to quickly see the distance traveled.

Reviewer: ```suggestion
This functionality allows you to quickly and temporarily see the **Odometer** screen when you are on any other screen.
If this option is activated, whenever you are on a screen other than **Odometer**, holding down the upper or lower button on the remote will temporarily switch to the **Odometer** display. Releasing the button on the controller will return to the screen you were previously.
This is especially useful for sections where you need to pay special attention to the **Heading** or **Speed ​​**, but from time to time you need to quickly see the distance traveled.
```

### Invert Screen

Allows you to invert the screen image 180º.

> This is useful in case you have to install the **Baja Pro** in an inverted position.

Reviewer: ```suggestion
This is useful in case you have to install the **Open Rally Computer** in an inverted position.
```

### GPS data

> Raw data from the GPS module among which you can find Latitude, Longitude, Height and plus.

Reviewer: ```suggestion
Raw data from the GPS module among which you can find Latitude, Longitude, Height and more.
```

This could be useful in the event that you need to share your current location with somebody.

### Update firmware

> Update the **Baja Pro** firmware to the latest version.

Reviewer: ```suggestion
Update the **Open Rally Computer** firmware to the latest version.
```

> The description of this functionality will be updated in this manual as soon as a firmware update.

Reviewer: ```suggestion
The description of this functionality will be updated in this manual as soon as a firmware update is released.
```

### Version

> It shows the serial number of your Baja Pro and the current version of the firmware installed in the device.

Reviewer: ```suggestion
It shows the serial number of your device and its current firmware version.
```

### Factory settings

> Restores all the **Baja Pro** settings and settings to their factory state.

Reviewer: ```suggestion
Restores the **Open Rally Computer** values and settings to their factory state.
```
